meetings/2018-02-01.md

```diff
@@ -66,7 +66,7 @@
 
 ### Decision structure (consensus, etc.)
 
-* There is a Foundation marketing committee. Check with Greg (gwallace@linuxfoundation.org)
+* There is a Foundation marketing committee. Check with [Greg](mailto:gwallace@linuxfoundation.org)
```

Comment:
This was an editorial change, since it doesn't like using the angle brackets around the email addresses